plugins/ptp_operator/metrics/logparser.go

```diff
@@ -519,6 +531,36 @@ func (p *PTPEventManager) ParseGNSSLogs(processName, configName, output string,
 	}
 }
 
+// extractPTPHaMetrics ... parse logs for ptp ha
+func extractPTPHaMetrics(processName, output string) (profile string, state int64, err error) {
+	// phc2sys[1710435400]:[phc2sys.2.config] ptp_ha_profile profile1 state 1
+	index := FindInPhc2SysLogForCfgFileIndex(output)
+	state = -1
+	if index == -1 {
+		log.Errorf("config name is not found in log outpt")
```

Comment:
print out content of output to help debug the error.

Reply:
done